decisions_test.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"fmt"
+	"testing"
+)
+
+/// Running simple tests on the choose function from the decisions trigger
+func TestDecisionsChoose(t *testing.T) {
+	var tests = []struct {
+		input           string
+		possibleOutputs []string // This makes the check efficient
+	}{
```

Comment:
we might be able to reuse this struct for other tests

Reply:
True, but while we don't use it in different tests, I don't see the point of having one more struct in the code.
It might be useful to have a separate test package to have all sorts of structs.